techdocs/script/cli.py

```diff
@@ -1,18 +1,19 @@
-import sys
 import argparse
+import os
+import sys
 
+from config import ConfigError, ConfigLoader
 from copier import Copier, Executor, PrintingExecutor
-from config import ConfigLoader, ConfigError
 from filesystem import Filesystem
-from operations import Detector
-
+from operations import CopyDetector, DeleteDetector, IndexLoader
 
 if __name__ == "__main__":
     fs = Filesystem()
     parser = argparse.ArgumentParser(
         epilog="""A program, that copies files from one directory to another, based on a configuration file."""
     )
     parser.add_argument("command", choices=["copy"])
+    parser.add_argument("--repo", dest="repo", required=True)
```

Comment:
> Since it's strictly for indexing purposes and repository has nothing to do with it actually.

Maybe `--index-as` / `--index-key` / `--index` then?